aws/coreos-docker-servers.tf

```diff
@@ -30,3 +33,23 @@ resource "template_file" "etcd_discovery_url" {
     }
   }
 }
+
+resource "aws_security_group" "docker_node" {
```

Comment:
Yes, for instance. 

But I am not sure if it is worth it, because will prevent access from the router to the docker node. On one hand makes lots of sense, as the docker daemon is the most critical service there and if somebody manages to trick the router to hit the docker node we are gone. 

But on the other hand maybe we should secure the docker node in a different way, because a container could access the local docker daemon anyway. A more secure approach can be for instance use a different network interface for docker nodes, use ssh tunnels, or similar.